vendor/github.com/containers/virtcontainers/hypervisor.go

```diff
@@ -136,6 +143,30 @@ func (conf *HypervisorConfig) valid() (bool, error) {
 		return false, fmt.Errorf("Missing image path")
 	}
 
+	if conf.Unconstrained_sockets == 0 {
+		return false, fmt.Errorf("Missing cpu sockets")
```

Comment:
I think it might be preferable to not require the specification of sockets, cores and threads and have virtcontainers default to sensible values itself.